gdcdictionary/schemas/molecular_test.yaml

```diff
@@ -3165,6 +3169,9 @@ properties:
       THRAP3:
         $ref:
           - "_terms_enum.yaml#/thrap3/common"
+      TLR2:
+        $ref:
+          - "_terms_enum.yaml#/tlr2/common"
```

Comment:
@tlicht3 Would it be okay to remove this?  I don't think we have it in the enum terms file.

Reply:
Removed!